product_supplierinfo_for_customer/__openerp__.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Use product supplier info for customers too",
+    "version": "9.0.1.0.0",
+    "author": "AvanzOSC, "
+              "Tecnativa, "
+              "Eficent, "
+              "Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/product-attribute",
+    "category": "Sales Management",
+    "license": 'AGPL-3',
+    "depends": [
+        "base",
```

Comment:
do you really need this?

Reply:
I think so, because it's inheriting from res.partner

Reply:
product is already adding the dependency IMO.
Not hurting but I usually prefer to have the strictly necessary